internal/command/volumes/extend.go

```diff
@@ -72,7 +73,22 @@ func runExtend(ctx context.Context) error {
 		return err
 	}
 
-	sizeGB := flag.GetInt(ctx, "size")
+	sizeFlag := flag.GetString(ctx, "size")
+	sizeBytes, err := units.FromHumanSize(sizeFlag)
```

Comment:
do we have to change it to `RAMInBytes()` to match expectations?

Reply:
Since this only accepts sizeGB, not necessary I think.

Reply:
hmm, if we ever accept 1024GB as volume size that is going to expose the same issue you found with `fly scale memory` command and 1024MB.

Reply:
Even if we accept 1024GB, it will be divided by a billion and we still get 1024GB.

Reply:
Fair point. I guess the same applies to `fly scale memory`, no need to use `RAMInBytes` as long as the parsed value is divided by `units.MB` instead of `units.MiB`

Reply:
If I pass `1gb`, `FromHumanSize` returns `10^9` and whichever way I divide it, I can't get to 1024MB.

Reply:
I could pass a parser func to `helpers.ParseSize` though if we still want to reuse the logic:

`helpers.ParseSize(sizeFlag, units.RAMInBytes, units.MiB)`
`helpers.ParseSize(sizeFlag, units.FromHumanSize, units.GB)`

Reply:
Done